src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -99,7 +99,7 @@ public function setHandler(callable $handler = null): ?callable
      *
      * @return string The previous file link format
      */
-    public function setFileLinkFormat($fileLinkFormat): string
+    public function setFileLinkFormat($fileLinkFormat): ?string
```

Comment:
please remove the return type because it can also be a `FileLinkFormatter` (see `@param`) and adjust the annotation to be `@return string|FileLinkFormatter|null`